experiments/ClimaEarth/run_amip.jl

```diff
@@ -993,19 +1032,19 @@ if ClimaComms.iamroot(comms_ctx)
         end
     end
 
-    ## ci plots
+    ## plot extra atmosphere diagnostics if specified
     if config_dict["ci_plots"]
         @info "Generating CI plots"
         include("user_io/ci_plots.jl")
         make_plots(Val(:general_ci_plots), [atmos_sim.integrator.p.output_dir], dir_paths.artifacts)
     end
 
     ## plot all model states and coupler fields (useful for debugging)
-    !(comms_ctx isa ClimaComms.MPICommsContext) && debug(cs, dir_paths.artifacts)
+    !CA.is_distributed(comms_ctx) && debug(cs, dir_paths.artifacts)
 
-    if isinteractive()
-        ## clean up for interactive runs, retain all output otherwise
-        rm(dir_paths.output; recursive = true, force = true)
-    end
+    if isinteractive() #hide
+        ## clean up for interactive runs, retain all output otherwise #hide
+        rm(dir_paths.output; recursive = true, force = true) #hide
```

Comment:
Just a reminder that we should not do this (it's very dangerous)

Reply:
Why is this very dangerous? It's only being done for interactive runs

Reply:
If you set `COUPLER_OUTPUT_DIR` to be your home directory, this line would remove all your files. Or, if you set `COUPLER_OUTPUT_DIR = "."`, this line will remove your working folder.

We should never remove files and folders we haven't created.